providers/dns/cloudflare/cloudflare.go

```diff
@@ -57,17 +70,25 @@ func NewDNSProvider() (*DNSProvider, error) {
 	if err != nil {
 		var errT error
 		values, errT = env.GetWithFallback(
-			[]string{"CLOUDFLARE_API_TOKEN", "CF_API_TOKEN"},
+			[]string{"CLOUDFLARE_DNS_API_TOKEN", "CF_DNS_API_TOKEN"},
 		)
 		if errT != nil {
 			return nil, fmt.Errorf("cloudflare: %v or %v", err, errT)
 		}
+		if z := env.GetOrFile("CLOUDFLARE_ZONE_API_TOKEN"); z != "" {
+			values["CLOUDFLARE_ZONE_API_TOKEN"] = z
+		} else if z = env.GetOrFile("CF_ZONE_API_TOKEN"); z != "" {
+			values["CLOUDFLARE_ZONE_API_TOKEN"] = z
+		} else {
+			values["CLOUDFLARE_ZONE_API_TOKEN"] = values["CLOUDFLARE_DNS_API_TOKEN"]
+		}
```

Comment:
@ldez This is more what I had in mind when I previously used the `env.GetWithFallback` cascade:

1. try `CLOUDFLARE_ZONE_API_TOKEN`
2. if that's empty, try its alias `CF_ZONE_API_TOKEN`
3. it that's empty as well, fall back to `CLOUDFLARE_DNS_API_TOKEN`

Is there a better suited method in the `env` package, I just have not seen?

Reply:
Seems too specific for now to create a method in the `env` package.